recipes/scikit-time/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scikit-time" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: adb3199bb9084689999196185143027d4ef57e5eaee97ed6bf7c9a4193cb702e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py27 or osx or win] 
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - cython
+    - numpy
+    - scipy
```

Comment:
Why do you need scipy in the host section? Is it used during the build of the package?

Reply:
exactly, we build some extension against the cython headers of blas provided by scipy.